java/gast_to_code/gast_to_code_java.py

```diff
@@ -31,23 +32,23 @@ def handle_log_statement(gast):
     def handle_var_assign(gast):
         var_id = router.gast_to_code(gast["varId"], "java")
         var_value = router.gast_to_code(gast["varValue"], "java")
-        var_type = gast["varValue"]["type"]
         
-        if var_type == "num":
-            kind = "int"
-        elif var_type == "str":
-            kind = "String"
-        elif var_type == "bool":
-            kind = "boolean"
-        else:
-            kind = "customType"
+        kind = java_helpers.gast_to_java_type(gast["varValue"])
  
         return kind + " " + var_id + " = " + var_value
 
     def handle_aug_assign(gast):
         pass
 
+    # FIXME (taiga#149) gast_to_code should not be able to return System.out.println(1, 2)
     def handle_func_call(gast):
+        # handles logstatement for single array
+        if gast["value"]["type"] == "logStatement" and len(gast["args"]) == 1 and gast["args"][0]["type"] == "arr":
```

Comment:
Any reason we can't support multiple arguments?

Reply:
Multiple arguments not supported for java: https://stackoverflow.com/questions/25451719/java-system-out-println-syntax-help-for-multiple-variables/25451829